<launch>
	<arg name="marker_size" default="4.4" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/camera/depth_registered/points" />
	<arg name="cam_info_topic" default="/camera/rgb/camera_info" />		
	<arg name="output_frame" default="/camera_rgb_optical_frame" /> <!-- name of the frame that the published cartesian locations of the ar tages will be relative to...so this should chane -->

	<include file="$(find openni_launch)/launch/openni.launch"/>  
	
    <group ns="/kinect">

	<node name="kinect_ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)">  

	<remap from="vizualization_marker" to="kinect_vizualization_marker" />
	<remap from="ARmarker_points" to="kinect_ARmarker_points" />
	<remap from="ARmarker_points_array" to="kinect_ARmarker_points_array" />
	<remap from="ar_pose_marker" to="kinect_ar_pose_marker" />

	</node>

	</group>

</launch>
